cli/unregister/unregister.go

```diff
@@ -28,10 +28,14 @@ type ApiAttributes struct {
 }
 
 // DoIt unregisters this Horizon edge node and resets it so it can be registered again
-func DoIt(forceUnregister, removeNodeUnregister bool, deepClean bool, timeout int) {
+func DoIt(forceUnregister, removeNodeUnregister bool, deepClean bool, timeout int, container bool) {
 	// get message printer
 	msgPrinter := i18n.GetMessagePrinter()
 
+	if !deepClean && container {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, "Cannot use -C flag if not performing a deep clean. Must specify -D flag to use -C flag.")
```

Comment:
Please use `msgPrinter.Sprintf("Cannot use -C ...") for the message for globalization.